### ISSUE TITLE
TOM Model Output Not as Expected

### ISSUE THREAD

**Issue by jsanket9578** [shionhonda/viton-gan#6]
I am trying to implement VITON GAN model using the same dataset as CP-VTON and pre-trained model files available [here](https://drive.google.com/drive/folders/1Zcc55A1w6bUNG4cMYnmsIPvOJWvBsmDH).

The output of the GMM model (warped cloth image and mask) looks good but the final try-on output of TOM model exhibits large error. Specifically, the "rendered_person" image (7th picture in the sample image attached) which is the output of the UNET network has very weird colors and in some cases, the shape of the target cloth is also haphazard; even though the GMM output for the same is appropriate.

You can find sample output of our model [here](https://drive.google.com/drive/folders/1IKT5pvT2SCycIRAKphWTKZU1XQcobXbK). The model was only tested on train data as of now.
We wanted to understand what is the exact problem.

![000007_1](https://user-images.githubusercontent.com/55049530/70266092-d8b3a400-17c1-11ea-8190-8c92334d6398.jpg)

**Comment by shionhonda** [comment]
Thanks for reporting.
I'm not sure what the exact problem is, but it happens sometimes, as shown in "Failure cases" in the paper.
Nevertheless, it is weird that it also happens with the training data.
